include/pybind11/embed.h

```diff
@@ -198,9 +198,14 @@ inline void initialize_interpreter(bool init_signal_handlers = true,
         init_signal_handlers, argc, argv, add_program_dir_to_path);
 #else
     PyConfig config;
-    PyConfig_InitIsolatedConfig(&config);
-    config.isolated = 0;
-    config.use_environment = 1;
+    PyConfig_InitPythonConfig(&config);
+    config.dev_mode = 0;
+    config.safe_path = 1;
+    config.configure_c_stdio = 0;
+    config.faulthandler = 0;
```

Comment:
Can't faulthandler also be controlled by environment variable though? Should we really zero-out all these options like so?

Reply:
Sorry, I changed this to Draft while I attempt to figure out an optimal config